### PR TITLE
fleet: add doc for policy management

### PR DESCRIPTION
**Pull request by zirain**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/content/en/docs/fleet-manager/policy.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+weight: 30
+description: >
+  The easiest way to enable policy management with fleet.
+---
+
+In this tutorial we’ll cover the basics of how to use [Fleet](https://kurator.dev/docs/references/fleet-api/#fleet) to manage policy on a group of clusters.
+
+## Architecture
+
+Fleet's multi cluster monitoring is built on top [Kyverno](https://kyverno.io/),
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/policy.md:13]: s/monitoring/policy management

File: docs/content/en/docs/fleet-manager/policy.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+title: "Enable Policy Management with fleet"
+linkTitle: "Policy Management"
+weight: 30
+description: >
+  The easiest way to enable policy management with fleet.
+---
+
+In this tutorial we’ll cover the basics of how to use [Fleet](https://kurator.dev/docs/references/fleet-api/#fleet) to manage policy on a group of clusters.
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/policy.md:9]: policies

File: docs/content/en/docs/fleet-manager/policy.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+## Architecture
+
+Fleet's multi cluster monitoring is built on top [Kyverno](https://kyverno.io/), 
+
+## Prerequisites
+
+1. Setup Fleet manager by following the instructions in the [installation guide](/docs/setup/install-fleet-manager/).
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/policy.md:17]:
```suggestion
1. Setup Fleet manager following the instructions in the [installation guide](/docs/setup/install-fleet-manager/).
```

File: docs/content/en/docs/fleet-manager/policy.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+1. Fleet manager depends on [Fluxcd](https://fluxcd.io/flux/), setup by following the instructions in the [installation guide](/docs/setup/install-fluxcd/).
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/policy.md:19]: FluxCD

Comment (hzxuzhonghu) [reply docs/content/en/docs/fleet-manager/policy.md]: We should move it to /docs/setup/install-fleet-manager/

File: docs/content/en/docs/fleet-manager/policy.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+kubectl apply -f examples/fleet/policy/kyverno.yaml
+```
+
+After a while, we can see the fleet is `ready`:
+
+```console
+kubectl wait fleet quickstart --for='jsonpath='{.status.phase}'=Ready'
+```
+
+### Veify pod security policy
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/policy.md:36]: Verify

File: docs/content/en/docs/fleet-manager/policy.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+## Architecture
+
+Fleet's multi cluster policy management is built on top [Kyverno](https://kyverno.io/), 
+
+## Prerequisites
+
+1. Setup Fleet manager following the instructions in the [installation guide](/docs/setup/install-fleet-manager/).
+
+2. Fleet manager depends on [FluxCD](https://fluxcd.io/flux/), setup by following the instructions in the [installation guide](/docs/setup/install-fluxcd/).
```
[the remark this answers is not in the document]
Comment (zirain) [reply docs/content/en/docs/fleet-manager/policy.md:19]: will do it in a seprated PR

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**